### PR TITLE
Fix security issues

### PR DESCRIPTION
**Pull request by DmitryAvramec**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -281,4 +281,14 @@ def cors_preflight_check
...
     end
   end
+
+  def find_class(asset)
+    classes = ActiveRecord::Base.connection.tables.map(&:classify)
+    find = classes.find { |m| m == asset.classify }
+    if find
+      find.safe_constantize
+    else
+      raise "Unknown resourse"
```
Comment (CloCkWeRX) [inline app/controllers/application_controller.rb]: Spelling: resource

File: app/controllers/admin/fields_controller.rb [hunk 2]
```diff
@@ -114,7 +114,22 @@ def subform
   protected
 
   def field_params
-    params[:field].permit!
+    params.require(:field).permit(
```
Comment (CloCkWeRX) [inline app/controllers/admin/fields_controller.rb]: I *doubt* that anyone would have extended the field class while using it as an engine; but we should make sure we write a migration guide for anywhere we went for explicit param whitelisting, suggesting people will want to .permit() their customised attributes

**Comment by CloCkWeRX** [comment]
and rubocop --auto-gen-config too

**Comment by johnnyshields** [comment]
@DmitryAvramec there's still a rubocop error here.

**Pull request merged**